generator/templates/src/components/Component/docs/index.story.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { {%ComponentName%} } from '../';
+import * as examples from './examples';
+import {
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { baseScope } from '../../../../stories/utils/LiveCodeExample';
```

Comment:
```suggestion
import { LiveScope } from '../../../../stories/utils/allComponents';
```
no reason to proxy it through LiveCodeExample. it was an ad-hoc implementation in wsr

also, fyi, in future `code` section will not require to set `components` each time. it will be setup just once.